src/code/components/genome.js

```diff
@@ -8,17 +8,27 @@ import {getChromosomeDescriptor} from '../fv-components/fv-chromosome-image';
  * Usually defined by passing in a Biologica Organism, but may also be defined by
  * passing in a map of Biologica Chromosomes and a Biologica Species.
  */
-const GenomeView = ({org, className="", ChromosomeImageClass, chromosomes, species, userChangeableGenes=[], visibleGenes=[], hiddenAlleles=[], editable=true, showLabels=true, selectedChromosomes={}, small=false, orgName, displayStyle, onAlleleChange, onChromosomeSelected, chromosomeHeight}) => {
+const GenomeView = ({org, className="", ChromosomeImageClass, chromosomes, species, userChangeableGenes=[], visibleGenes=[], hiddenAlleles=[], editable=true, showLabels=true, labelEmptyChromosomes=false, selectedChromosomes={}, small=false, orgName, displayStyle, onAlleleChange, onChromosomeSelected, chromosomeHeight}) => {
   // Prefer explicitly passed variables to those extracted from the passed organism
   chromosomes = chromosomes || org.genetics.genotype.chromosomes;
+
+  let fullChromosomes;
+  if (labelEmptyChromosomes && org) {
+    fullChromosomes = org.genetics.genotype.chromosomes;
```

Comment:
This is how I'm passing all chromosomes to the genome during the flashing animation, even though some chromosomes passed to the `GenomeView` are empty while the animation plays.